girder/utility/webroot.py

```diff
@@ -88,9 +90,18 @@ def __init__(self, templatePath=None):
             'plugins': [],
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder'
+            'title': 'Girder',
+            'contactEmail': ModelImporter.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
         }
 
+        events.bind('model.setting.save.after', CoreEventHandler.WEBROOT_SETTING_CHANGE,
+                    self._onSettingSave)
+
+    def _onSettingSave(self, event):
```

Comment:
Indeed, as part of fully implementing #2274, the `WebrootBase.indexHtml` should be switched to use a proper cache. I see nothing more that can be done on this PR.